### PR TITLE
Docs creation

### PR DESCRIPTION
**Pull request by VQTran123**
Added folder for documentation as well as an rst document on how to set up MMS5-Layer1 Service on local machine.

### CODE REVIEW

**Comment by HuiJun** [comment]
We're going to want to delete that .DS_Store folder.

File: docs/mms5layer1setup.rst [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    cd deploy
+    export APP_URL="http://layer1-service"
+    npx ts-node src/main.ts $APP_URL > ../src/test/resources/cluster.trig
+
+In order to run the tests, you can return to your root directory and run the following commands.
+::
+    cd..
+    BUILDKIT=0 docker build --network=mms5-test-network -t mms5-test:latest -f Dockerfile-Test .
+    docker run --network=mms5-test-network --name mms5-test-container mms5-test:latest
+    SIG_INT=$?
```
Comment (HuiJun) [inline docs/mms5layer1setup.rst:23]: The SIG_INT variable isn't needed, unless we are specifically looking at the exit code.

File: docs/mms5layer1setup.rst [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    docker cp mms5-test-container:/application/build/reports/tests <directory you want to store results>/results
+    exit "$SIG_INT"
```
Comment (HuiJun) [inline docs/mms5layer1setup.rst:25]: We don't need to exit with the exit code here.

**Review by HuiJun — CHANGES_REQUESTED** [review]

**Review by VQTran123 — COMMENTED** [review]
Deleted .DS_Store and removed SIG_INT from document

**Review by HuiJun — APPROVED** [review]

**Pull request merged**